### PR TITLE
GUACAMOLE-1931: Document parameters for writing to existing recording files.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: src/configuring-guacamole.md [hunk 1]
```diff
@@ -2284,6 +2289,19 @@ help, the session will simply not be recorded.
...
+`recording-write-exiting`
+: If set to "true", instead of insisting on creation of a new file, and
+  appending numbers until a non-existing file is found, Guacamole will
+  simply write to the existing recording file, overwriting data that
+  may already be present. While overwriting recordings is not generally
+  desirable, this parameter is useful in situations where you want to
+  try to write to a named pipe, FIFO buffer, or other special device
+  that may be sending the recording data elsewhere.
+
+  This parameter only has an effect if graphical recording is enable. If
```
Comment (jmuehlner) [inline src/configuring-guacamole.md:2301]: Should be "enabled".

File: src/configuring-guacamole.md [hunk 2]
```diff
@@ -2348,6 +2370,20 @@ help, the session will simply not be recorded.
...
+: If this parameter is set to "true", instead of attempting to generate
+  unique file names by appending incremental numbers to the typescript name,
+  Guacamole will be allowed to write to an existing typescript file,
+  potentially ovewriting any data in that file. While this is not
+  desirable in most cases, it may be necessary in situations where the
+  typescript recording is being sent to a named pipe, FIFO buffer, or
+  other special device that is processing the data and sending it
+  elsewhere.
+
+  This parameter only has an effect if typescript recording is enable. If
```
Comment (jmuehlner) [inline src/configuring-guacamole.md:2383]: Same thing here - "enable" should be "enabled".

File: src/configuring-guacamole.md [hunk 3]
```diff
@@ -2348,6 +2370,20 @@ help, the session will simply not be recorded.
...
+  the `typescript-path` is not specified, recording of typescripts will be
+  disabled, and this parameter will be ignore.
```
Comment (jmuehlner) [inline src/configuring-guacamole.md:2385]: This should be "ignored".

File: src/configuring-guacamole.md (hunk 1 above)
Comment (necouchman) [reply src/configuring-guacamole.md]: Fixed via rebase.

File: src/configuring-guacamole.md (hunk 2 above)
Comment (necouchman) [reply src/configuring-guacamole.md]: Fixed be rebase.

File: src/configuring-guacamole.md (hunk 3 above)
Comment (necouchman) [reply src/configuring-guacamole.md]: Fixed via rebase.

**Pull request merged**